13.0 website login redirect back (#619)
itpp-labs/website-addons: website_login_redirect_back/README.rst

.. image:: https://itpp.dev/images/infinity-readme.png
   :alt: Tested and maintained by IT Projects Labs
   :target: https://itpp.dev

=============================
 Website Login Redirect Back
=============================

By default Odoo redirects portal users on `/my` portal dashboard after their success login.
This module makes redirection to the same page from where logging in process were initiated.

No special settings is needed, just install the module.

Typical usage of the module.
============================

> Say you are on the `/blog` page and click `Sing in` in the caption of the page to logging in.

Reviewer: ```suggestion
For example, you are on the `/blog` page and click `Sign in` button to log in.
```
